spec/smoke/find/course_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "nokogiri"
+require "spec_helper_smoke"
+
+class HtmlParserIncluded < HTTParty::Parser
```

Comment:
Can we break the class per file please

Reply:
This is so specific to this test that i don't see it adding value having it in its own file?